### ISSUE TITLE
Add --peers option for add / install command

### ISSUE THREAD

**Issue by emahuni** [pnpm/pnpm#5284]
**Describe the user story**

The peer deps feature is one of the cool features about this pkg manager. I think you should make it easier to make the tool install peer deps on cli without setting the config.

**Describe the solution you'd like**

allow 
`pnpm -i --peers`
and 
`pnpm add --peers package-to-add`

**Describe the drawbacks of your solution**

I don't know of any

**Describe alternatives you've considered**

none

**Comment by zkochan** [comment]
It is going to be inconsistent if you don't set the setting. Someone else removes the lockfile, runs install and the peers won't be installed.

**Comment by emahuni** [comment]
Yes I thought of that, but shouldn't that be the user's choice? If they want consistency, then there's a setting for that, they have to set it in the config files 🤷🏾‍♂️.

**Comment by emahuni** [comment]
The last command I proposed will only install peer deps in that package and it's descendents.

**Comment by zkochan** [comment]
I don't think complicating peer dependency installation with selective auto installation is worth it.

**Comment by BryanCrotazGivEnergy** [comment]
In CI I want to turn **auto-install-peers** off, so this cli setting would be useful

**Comment by zkochan** [comment]
This is implemented in current pnpm, so closing.

`autoInstallPeers` is on by default, so peers are installed without any flag, and `autoInstallPeers: false` turns it off.

If you still hit this on the latest pnpm, please comment with a reproduction and we will reopen.

---
Written by an agent (Claude Code, claude-opus-5).
